CHANGELOG.md

```diff
@@ -3,6 +3,24 @@ All notable changes to this project will be documented in this file.
 
 This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html)
 
+## [vector-0.40.1] - 2025-02-24
+
+### Releasing
+
+#### Features
+
+- Introduce release script (#446) ([2e252e2](https://github.com/vectordotdev/helm-charts/commit/2e252e20bfcb560847481b02e7d484f6302b48ad))
+
+### Vector
+
+#### Bug Fixes
+
+- Use `.KubeVersion.Version` rather than `.KubeVersion.GitVersion` for capability detection (#449) ([931fa10](https://github.com/vectordotdev/helm-charts/commit/931fa10afe7b04cf7e32a65768196a7bdf02a01e))
+
+#### Features
+
+- Regenerate CHANGELOG for vector-0.40.0 (#445) ([4b188a5](https://github.com/vectordotdev/helm-charts/commit/4b188a5e92966a1cf14b6b046541694b082e6df4))
```

Comment:
I think this is here because the commit uses `feat` but I think it is a `chore`, no?